FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.h

```diff
@@ -35,6 +35,15 @@ typedef NS_ENUM(NSInteger, FPRApplicationState) {
   FPRApplicationStateBackground,
 };
 
+/** Different modes of prewarm-detection */
+typedef NS_ENUM(NSInteger, PrewarmDetectionMode) {
+  DropAllEvents = 0,
+  OnlyActivePrewarm = 1,
+  OnlyDoubleDispatch = 2,
+  EitherActivePrewarmOrDoubleDispatch = 3,
+  KeepAllEvents = 4
```

Comment:
Under what scenario, we will use `KeepAllEvents`? Does it mean that we will include all prewarmed events?

Reply:
Yes, it means all app start are included, essentially reverting back to pre-iOS15 SDK. There's not much scenario I suppose where this would be used, probably only if Apple completely reverts prewarming feature. This is there just for completion so RC flag covers all possibilities.

Reply:
Typically Enums have prefixes for better understanding:

For example: For the above case, my recommendation of Enums would be:

```
typedef NS_ENUM(NSInteger, PrewarmDetectionMode) {
  PrewarmDetectionModeNone = 0,
  PrewarmDetectionModeActivePrewarm = 1,
  PrewarmDetectionModeDoubleDispatch = 2,
  PrewarmDetectionModeActivePrewarmOrDoubleDispatch = 3,
  PrewarmDetectionModeAll = 4
}
```

Reply:
Done